pkg/planner/core/expression_rewriter.go

```diff
@@ -2640,6 +2641,185 @@ func hasCurrentDatetimeDefault(col *model.ColumnInfo) bool {
 	return strings.ToLower(x) == ast.CurrentTimestamp
 }
 
+func encodeHandleFromRow(
+	ctx expression.EvalContext,
+	isVer infoschemactx.MetaOnlyInfoSchema,
+	args []expression.Expression,
+	row chunk.Row,
+) ([]byte, bool, error) {
+	dbName, isNull, err := args[0].EvalString(ctx, row)
+	if err != nil || isNull {
+		return nil, isNull, err
+	}
+	tblName, isNull, err := args[1].EvalString(ctx, row)
+	if err != nil || isNull {
+		return nil, isNull, err
+	}
+	is := isVer.(infoschema.InfoSchema)
+	if is == nil {
+		return nil, false, errors.New("missing information schema")
```

Comment:
```suggestion
	is, ok := isVer.(infoschema.InfoSchema)
	if !ok {
		return nil, false, errors.New("missing information schema")
```

the cast will crash if `isVer` is actually nil 

```
panic: interface conversion: interface is nil, not infoschema.InfoSchema
```

Reply:
It cannot be nil, I removed the nil-check.